www/js/survey/multilabel/confirmHelper.ts

```diff
@@ -71,9 +72,13 @@ export function getLabelInputDetails(appConfigParam?) {
 export function labelInputDetailsForTrip(userInputForTrip, appConfigParam?) {
   if (appConfigParam) appConfig = appConfigParam;
   if (appConfig.intro.mode_studied) {
-    if (userInputForTrip?.['MODE']?.data?.label == appConfig.intro.mode_studied) {
-      logDebug(`Found trip labeled with ${userInputForTrip?.['MODE']?.data?.label}, mode of study = ${appConfig.intro.mode_studied}.
-        Needs REPLACED_MODE`);
+    const modesStudied =
+      typeof appConfig.intro.mode_studied == 'string'
+        ? [appConfig.intro.mode_studied]
+        : appConfig.intro.mode_studied;
```

Comment:
we can remove this code once all the existing programs with a single mode of interest have been archived, right?

Can we add a TODO: here to flag that?

Reply:
If the long-term strategy is for this to always be an array, then yes. But in that case, it may be more appropriately named as `modes_studied`
And we'll need to change https://github.com/e-mission/nrel-openpath-deploy-configs/blob/38b12c8f8334d3156e70390b0f15ab92ef4eb427/.github/actions/convertIssue/parse-issue-body.js#L299

My thought was that we'd just keep `mode_studied` as `string | string[]` since the canonical program has one mode of study, and supporting multiple modes of study is a secondary use case